scripts/upgrade

```diff
@@ -16,10 +16,10 @@ set -e
 app=$YNH_APP_INSTANCE_NAME
 
 # Retrieve app settings
-domain=$(sudo yunohost app setting $app domain)
-path=$(sudo yunohost app setting $app path)
-admin=$(sudo yunohost app setting $app admin)
-is_public=$(sudo yunohost app setting $app is_public)
+domain=$(sudo ynh_app_setting_get $app domain)
+path=$(sudo ynh_app_setting_get $app path)
```

Comment:
you don't need (and can't) use `sudo` before calling helpers. It makes upgrade fail.